### DISCUSSION TITLE
Daily Standup - Thursday, April 27

### DISCUSSION THREAD

**Discussion by antonieto** [antonieto/ibm-dashboard#147] [General]
Share your daily standup update with the team here. Respond to this
discussion before the standup starts in the following format:

```
**What did you achieve in the last 24 hours?**:
 - 

**What are your priorities for the next 24 hours?**:
 - 

**Blockers**:
 -

**Shoutouts**:
 - @username for x
```

**Comment by antonieto** [comment]
**What did you achieve in the last 24 hours?**:
 - Started integrating tRPC into our project

**What are your priorities for the next 24 hours?**:
 - Decide our architecture (folder structure)
 - Specify that we are using _serverless_ architecture on our Architecture Diagram

**Blockers**:
 - I ll be traveling at 8 pm


**Shoutouts**:
 - @Jhoscar22

**Comment by DavBot02** [comment]
**What did you achieve in the last 24 hours?**:
 - Nothing really

**What are your priorities for the next 24 hours?**:
 - Work on folder structure in order to work on auth

**Blockers**:
 - Fedora petó

**Shoutouts**:
 - @Int-Angel jijiji

**Comment by Int-Angel** [comment]
**What did you achieve in the last 24 hours?**:
 - dont remember

**What are your priorities for the next 24 hours?**:
 - finishi the app

**Blockers**:
 - nothing can stop me

**Shoutouts**:
 - @DavBot02 jojojo
